elastic/eui
# [Docs] Remove GitHub pages hosted docs

## Summary

Now that Buildkite's release automation and caching is working nicely, we can remove our GitHub-hosted docs entirely in favor of https://eui.elastic.co.

## QA

- Confirm https://elastic.github.io/eui/#/ redirects to eui.elastic.co
- Confirm routing forwards work:
    - https://elastic.github.io/eui/#/guidelines/getting-started#setting-up-your-application
    - https://elastic.github.io/eui/#/templates/page-template/examples/sidebar

### General checklist

- N/A, docs only
